agent/config/config.go

```diff
@@ -65,9 +65,10 @@ type Agent struct {
 	// Consul is used to configure a consul API client
 	Consul *Consul `hcl:"consul,block"`
 
-	APMs       []*Plugin `hcl:"apm,block"`
-	Targets    []*Plugin `hcl:"target,block"`
-	Strategies []*Plugin `hcl:"strategy,block"`
+	PolicySources []*PolicySource `hcl:"source,block"`
```

Comment:
`source` feels ambiguous when considering the agent already includes a `policy` entry. I think having `source` under the `policy` block would make more sense.

```hcl
policy {
  dir = "/opt/policies"
  
  source "nomad" {
    enabled = false
  } 
}
```

Reply:
Good point, I will make this change.